handlers/accesscontrol/access-control.go

```diff
@@ -452,6 +584,20 @@ func (g *GateClient) QueryGate(body []byte) (bool, GateConfig, error) {
 			}
 			refreshInterval = int32(refreshIntervalFloat64)
 		}
+		if ri, ok := result["viewer_limit_per_user"]; ok {
```

Comment:
Not a regression, but any idea why we parse the JSON manually here instead of using `json:"field_name"` tags?

Reply:
No, that was one of my comments in the PR description. I'll address if while removing the rate limit logic. In this Linear ticket: https://linear.app/livepeer/issue/ENG-2120/change-rate-limite-to-viewer-limit